src/jobflow_remote/cli/flow.py

```diff
@@ -88,31 +90,46 @@ def flows_list(
 
     db_sort: list[tuple[str, int]] = [(sort.value, 1 if reverse_sort else -1)]
 
-    with loading_spinner():
-        flows_info = jc.get_flows_info(
-            job_ids=job_id,
-            db_ids=db_id,
-            flow_ids=flow_id,
-            states=state,
-            start_date=start_date,
-            end_date=end_date,
-            name=name,
-            metadata=metadata,
-            locked=locked,
-            limit=max_results,
-            sort=db_sort,
-            full=verbosity > 0,
-        )
+    if count:
+        with loading_spinner():
+            n_flows = jc.count_flows(
+                job_ids=job_id,
+                db_ids=db_id,
+                flow_ids=flow_id,
+                states=state,
+                start_date=start_date,
+                end_date=end_date,
+                name=name,
+                metadata=metadata,
+                locked=locked,
+            )
+        out_console.print(f"Selected Flows: {n_flows}")
```

Comment:
```suggestion
        out_console.print(f"Number of Flows matching query: {n_flows}")
```
Not sure if it is more clear or less clear. What do you think ? I let you decide :)

Reply:
I used "Number of" I hope it is fine. We can always change at a later stage if it is reported that it is not clear